board/common/rootfs/usr/sbin/container

```diff
@@ -320,7 +320,7 @@ delete()
     fi
 
     # Should already be stopped, but if not ...
-    log "$name: should already be stopped, double checking ..."
+    # log "$name: should already be stopped, double checking ..."
```

Comment:
Should these be kept around or deleted?

Reply:
Ah good point, those were initially for the command-line use-case, but they are just confusing, so I'll go in and drop them instead. Thanks!